src/main.rs

```diff
@@ -13,6 +15,16 @@ const DEFAULT_NATS_HOST: &str = "localhost";
 const DEFAULT_NATS_PORT: &str = "4222";
 /// Default lattice prefix
 const DEFAULT_LATTICE_PREFIX: &str = "default";
+/// Default log level
+const DEFAULT_LOG_LEVEL: &str = "info";
+/// Default allow loading components from the file system
+const DEFAULT_ALLOW_FILE_LOAD: &str = "false";
+/// Default provider shutdown delay
+const DEFAULT_PROV_SHUTDOWN_DELAY_MS: &str = "300";
```

Comment:
Why not https://doc.rust-lang.org/nightly/core/time/struct.Duration.html ?

Reply:
This one I will have to keep as a &str because of how clap parses args, but the resulting flag will be a Duration